host.py

```diff
@@ -250,11 +251,16 @@ def ssh_connect_looped(self, logins: List[Login]) -> None:
                     self._host = e.login()
                     return
                 except ssh_exception.AuthenticationException as e:
-                    logger.info(type(e))
-                    raise e
+                    logger.debug(type(e))
                 except Exception as e:
-                    logger.info(type(e))
-                    time.sleep(10)
+                    logger.debug(type(e))
+                    raise e
```

Comment:
This is tricky. I don't agree we should raise here.

Reply:
I think this is a bug currently.

i.e. I do not have keys set up, but I have the username:password for my host.

I try to connect with:

h = host.Host("myhostname")
h.ssh_connect("username", "password")

I get an exception that the rsa key login failed.

Reply:
Aahh I think I see what you mean. We will just raise if we get out of this for loop without returning

Reply:
should we ever get out? There is a reason why it was a "retry forever" loop